src/repository.py

```diff
@@ -216,13 +216,17 @@ def with_branch(self, branch_name: str) -> Iterator["Client"]:
         finally:
             self.switch(current_branch)
 
-    def get_summary(self) -> DiffSummary:
+    def get_summary(self, folder: str | None = DOCUMENTATION_FOLDER_NAME) -> DiffSummary:
```

Comment:
Nitpick, could we call it `directory`? I think that is the term used elsewhere in this repo